src/plugins/gui/amountQuotePlugin.ts

```diff
@@ -424,3 +527,33 @@ export const getBestQuote = (quotes: FiatProviderQuote[], priorityArray: Priorit
     if (bestQuote != null) return bestQuote
   }
 }
+
+const evm = '0x0d73358506663d484945BA85D0cd435ad610B0A0'
```

Comment:
Is this just some arbitrary address? Could use some documentation.

Also, move to top with other consts in caps